chain-vote/src/tally.rs

```diff
@@ -1,6 +1,6 @@
 use crate::{
     committee::*,
-    cryptography::{Ciphertext, CorrectElGamalDecrZkp},
+    cryptography::{Ciphertext, CorrectShareGenerationZkp},
```

Comment:
I thought we tried not to tie too much the stuff in cryptography with vote-specific domain (e.g. `UnitVectorProof` instead of `CorrectVoteZkp`). I don't have a strong opinion for this, but I would like to be consistent across proofs

Reply:
I agree. But `CorrectShareGenerationZkp` is not vote-specific. It is specific to threshold decryption, so I think it should be fine (note that there is a difference between `CorrectElGamalDecrZkp` and `CorrectShareGenerationZkp` in the proof itself, not only in the naming).